modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -342,12 +343,21 @@ export default {
     },
     getFollowingValues(item) {
```

Comment:
* Shouldn't there be code sharing on this between object and array via the mixin?
* Is it tested for both inline and non-inline arrays?

Reply:
Testbed does test both inline and non-inline arrays in different depts.
I've went for completely new mixin, as it makes sense only for fields having their own schema.